packages/exemplar-storybook-react/package.json

```diff
@@ -39,24 +39,24 @@
     "@storybook/react": "^5.1.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.2.2",
+    "@babel/core": "^7.4.5",
     "@babel/plugin-syntax-dynamic-import": "^7.2.0",
-    "@storybook/addons": "^5.1.0",
+    "@storybook/addons": "^5.1.9",
     "@storybook/react": "^5.1.0",
-    "assume": "^2.1.0",
-    "babel-eslint": "^10.0.1",
-    "babel-loader": "^8.0.5",
+    "assume": "^2.2.0",
+    "babel-eslint": "^10.0.2",
+    "babel-loader": "^8.0.6",
     "eslint": "^5.14.1",
     "eslint-config-godaddy-react": "^4.1.0",
```

Comment:
we should get greenkeeper on here at some point

Reply:
I'd prefer renovate, but agreed.